CMMSAMLIdentityProvider/IdPLauncher.aspx

```diff
@@ -32,15 +32,15 @@
                         Service Provider URL
                     </td>
                     <td>
-                        <asp:TextBox ID="txtSPURL" runat="server" Text="http://localhost:40728/ServiceProviderDefault.aspx"></asp:TextBox>
```

Comment:
I think these should remain the same as they allow the user to default run everything locally for testing purposes. Also they are not CMM specific resources.